Tensile/TensileCreateLibrary.py

```diff
@@ -1264,53 +1505,48 @@ def TensileCreateLibrary():
         masterLibraries[architectureName] = deepcopy(newLibrary)
         masterLibraries[architectureName].version = args.version
 
-  # create solution writer and kernel writer
-  kernels = []
-  kernelHelperOjbs = []
-  kernelHelperNames = set()
-
-  for solution in solutions:
-    solutionKernels = solution.getKernels()
-    for kernel in solutionKernels:
-      if kernel not in kernels:
-        kernels.append(kernel)
-    solutionHelperKernels = solution.getHelperKernelObjects()
-    for ko in solutionHelperKernels:
-      kname = ko.getKernelName()
-      if kname not in kernelHelperNames:
-        kernelHelperOjbs.append(ko)
-        kernelHelperNames.add(kname)
-
+#<<<<<<< HEAD
```

Comment:
Is this a merge conflict that's unresolved?

Reply:
Resolve in last commit